app/lib/finder_api.rb

```diff
@@ -116,6 +121,12 @@ def allowed_values_for_facet_details(facet_key, facet_details)
       .reject { |f| f["label"].blank? || f["value"].blank? }
   end
 
+  def allowed_values_from_registry(facet_key)
+    registries.all[facet_key].values
+      .map { |_, results| present_facet_option(results, facet_key) }
```

Comment:
Not sure if `_, ` is needed, or you can just do `map { |value| ... }`

Reply:
I'll double check.